src/components/DeviceDetails/__tests__/DeviceDetails.test.tsx

```diff
@@ -0,0 +1,37 @@
+import { omit } from "lodash";
+import { cleanup } from "@testing-library/react";
+import { render } from "@deskpro/app-testing-utils";
+import { DeviceDetails } from "../DeviceDetails";
+import { mockDevices } from "../../../../testing";
```

Comment:
Would be better to use alias like `@/testing/...`

Reply:
Done. Have been updated jest config